demos/src/demo.js

```diff
@@ -0,0 +1,3 @@
+import oLightSignup from '../../';
+
+oLightSignup.init();
```

Comment:
Likewise for consistency with the new name would it be worth renaming the demos?